tests/pytests/unit/modules/virt/test_domain.py

```diff
@@ -254,3 +254,66 @@ def test_get_disk_convert_volumes(make_mock_vm, make_mock_storage_pool):
                 "virtual size": 214748364800,
             },
         } == virt.get_disks("srv01")
+
+
+def test_update_stop_on_reboot_reset(make_mock_vm):
```

Comment:
Can we get some docstrings added to these tests?

Reply:
added :)